src/api/index.ts

```diff
@@ -17,10 +19,20 @@ export const updatePost = (id: string, updatedPost: IPostData) => API.patch('/po
 export const reactPost = (id: string, action: 'like' | 'dislike') => API.patch(`/posts/react?id=${id}&action=${action}`);
 export const deletePost = (id: string) => API.delete(`/posts/delete?id=${id}`);
 
+
+// Comments API Routes
+export const fetchComment = (commentId: string) => API.get('/comments/' + commentId);
+export const fetchCommentBySearch = (postId?: string, userId?: string) => API.get(`/comments/search?postId=${postId}&userId=${userId}`);
+export const createComment = (postId: string, comment: ICommentData) => API.post(`/comments/create?postId=${postId}`, comment);
+export const updateComment = (commentId: string, comment: ICommentData) => API.patch(`/comments/update`, { id: commentId, comment });
+export const reactComment = (commentId: string, action: 'like' | 'dislike') => API.patch(`/comments/react?commentId=${commentId}&action=${action}`);
+export const deleteComment = (commentId: string, postId: string) => API.delete(`/comments/delete?postId=${postId}&commentId=${commentId}`);
+
+
 // User API Routes
 export const logout = () => API.post('/user/logout');
 export const login = (loginData: ILogIn) => API.post('/user/login', loginData);
 export const signUp = (signupData: ISignUp) => API.post('/user/signup', signupData);
 export const getUser = (username: string) => API.get(`/user?username=${username}`);
 export const updateUser = (updateData: IUpdateUser) => API.patch('/user/update', updateData);
-export const checkAuthenticated = () => API.post("/user/check");
\ No newline at end of file
+export const checkAuthenticated = () => API.post('/user/check');
```

Comment:
I mean everything else is fine, but why this? I thought you liked double quoted strings.

Reply:
Because it was odd I've out, everything on front end is in single quotes.